RedHatInsights/compliance-frontend
# RHICOMPL-552 - Update rules step in policy wizard

This updates the "Rules" step in the policy creation wizard to further match the mockups.

![Kapture 2020-05-28 at 10 52 56](https://user-images.githubusercontent.com/7757/83120862-e0e5ff80-a0d1-11ea-9c06-755eb32c3a9f.gif)


**Test-protocol:**

1. Open the Policy page and the "Create Policy" wizard
2. Fill out steps 1&2 and continue to the "Rules" step.
3. Select or deselect rules
	- The rules count should update
	- A "Restore to default selection" link should appear
		- Clicking should update the selected rules to the default policy rules.
4. Continue to the "Systems" step and return to Rules
	* The rules should still match the previously selected ones
5. Return to the first step and change the policy type
6. Go forward to the "Rules" step
	* Description and default rules should match the newly selected policy type
7. Continue forward and finish the wizard to create a policy with the selected rules.
